templates/donors.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="donors">
+  <div class="donors_image">
+    {{{asset image}}}
+  </div>
+  <div class="donors_info">
+    <p class="donors_description">{{description}}</p>
+    <div class="donors_link_container">
+      <div class="donors_ctaPrimary_link">
+        {{{module ctaPrimary}}}
+      </div>
+      <div class="donors_ctaSecondary_link">
+        {{{module ctaSecondary}}}
+      </div>
+    </div>
+  </div>
+  <img class="donors_figure" src="/image/shape-m.svg" alt="shape of m"/>
```

Comment:
Since this is a decorative imgae alt text should be set as alt="" and a role:"presentation" attribute should be included in the html, to let screen reader know, this image should not be announced.

Reply:
thanks, i corrected it!!